### ISSUE TITLE
Raidboss: Timeline is not correct for Titan in The Navel(Hard)

### ISSUE THREAD

**Issue by oukenteikan** [quisquous/cactbot#4029]
## Description

Timeline is not correct for hard Titan in P4. Note, timeline is correct at the beginning of P4.
though i reproduced alone in the following example, this will also occur for a party level synced.

## Additional information

Titan is casting "怒震"×4, but the timeline is not there.

<img width="1687" alt="怒震" src="https://user-images.githubusercontent.com/33342734/149617346-f127fdd3-f1d8-4f1d-9aa1-d3f611bfea46.PNG">

**Comment by quisquous** [comment]
Do you have a network log for this?

**Comment by oukenteikan** [comment]
[Network_22105_20220205 - 副本.log](https://github.com/quisquous/cactbot/files/8007408/Network_22105_20220205.-.log)

**Comment by oukenteikan** [comment]
[network_log_for_hard_titan.log](https://github.com/quisquous/cactbot/files/8007414/network_log_for_hard_titan.log)

**Comment by quisquous** [comment]
Hmm, I guess our timeline testing utilities only work in English, so this is pretty tricky for me to try to debug.  By P4, do you mean after you break the heart? I'm pretty sure he opens with mountain buster into tumults after the heart.

**Comment by oukenteikan** [comment]
Yes, after the heart. I found it is actually Phase 5 on English ffxiv wiki, sorry. 
I have checked the video record (a small video is upload [here](https://user-images.githubusercontent.com/33342734/153162819-2febe66a-d3c9-4e47-93e7-032ad995eeb8.mp4)). It seems that every stomp in tumult delays the timeline for about 1s, so it looks like
-- Titan's first stomp, Weight of The Land in 7.5s -> 8.7s
about 1s later
-- Titan's second stomp,  Weight of The Land in 7.6s -> 8.7s 
about 1s later
-- Titan's third stomp, Weight of The Land in 7.5s -> 8.7s
about 1s later
-- Titan's fourth Stomp, no delay
about 7s later
-- Titan casts Weight of The Land but the timeline is delayed about 3s. 
This bug does not happen with the tumult in any previous phase.
